features/terms/list/build.gradle

```diff
@@ -49,8 +50,7 @@ dependencies {
     implementation project(":features:core:di")
 
     implementation project(":infra:core:repository")
-    implementation project(":infra:use-case")
 
-    implementation project(":foundation:models")
+    implementation project(":foundation:contracts")
```

Comment:
Since anything that uses `contracts` also needs `models`, I exposed `models` as part of the API of `contracts`. That's why it can be replaced here.

Note that the dependency on `:infra:use-case` is gone 🎉